libraries/chain/backing_store/db_key_value_format.cpp

```diff
@@ -5,6 +5,29 @@
 
 namespace eosio { namespace chain { namespace backing_store { namespace db_key_value_format {
    namespace detail {
+      constexpr std::size_t key_size(key_type kt) {
+         switch(kt) {
+            case key_type::primary:
+               return sizeof(uint64_t);
+            case key_type::sec_i64:
+               return sizeof(uint64_t);
+            case key_type::sec_i128:
+               return sizeof(eosio::chain::uint128_t);
+            case key_type::sec_i256:
+               return sizeof(key256_t);
+            case key_type::sec_double:
+               return sizeof(float64_t);
+            case key_type::sec_long_double:
+               return sizeof(float128_t);
+            case key_type::table:
+               return 0;
```

Comment:
Just wanted to make sure the size of key_type::table is 0.

Reply:
yes, and added a comment to hopefully make it clearer